src/blog/v8-release-84.md

```diff
@@ -0,0 +1,83 @@
+---
+title: 'V8 release v8.4'
+author: 'Camillo Bruni, '
```

Comment:
Please add author's tagline (see previous releases for inspiration).

Reply:
done.